python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -107,19 +107,30 @@ def launch_run(self, context: LaunchRunContext) -> None:
         # Run a task using the same network configuration as this processes's
         # task.
 
-        response = self.ecs.run_task(
-            taskDefinition=task_definition,
-            cluster=metadata.cluster,
-            overrides={"containerOverrides": [{"name": self.container_name, "command": command}]},
-            networkConfiguration={
+        ecs_run_task_args = {
+            "taskDefinition": task_definition,
+            "cluster": metadata.cluster,
+            "overrides": {
+                "containerOverrides": [{"name": self.container_name, "command": command}]
+            },
+            "networkConfiguration": {
                 "awsvpcConfiguration": {
                     "subnets": metadata.subnets,
                     "assignPublicIp": "ENABLED",
                     "securityGroups": metadata.security_groups,
                 }
             },
-            launchType="FARGATE",
-        )
+            "launchType": "FARGATE",
+        }
+
+        try:
```

Comment:
Can we also extend this test:

https://github.com/dagster-io/dagster/blob/c1292165f1daf7b747ffae60013188b69171eae2/python_modules/libraries/dagster-aws/dagster_aws_tests/ecs_tests/launcher_tests/test_launching.py#L86

To show that not only our stub mimics ECS correctly, but that we're interacting with our stub correctly? I'm thinking we add a few tags to our task definition, we configure our launcher to propagate tags, we launch a run, and we check that our tags from the task definition were copied over.